src/it/powermock/src/test/java/test/XTest.java

```diff
@@ -1,34 +0,0 @@
-package test;
-
-import jenkins.model.Jenkins;
-import static org.junit.Assert.assertEquals;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.Mock;
-import org.powermock.api.mockito.PowerMockito;
-import org.powermock.core.classloader.annotations.PowerMockIgnore;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
-
-@RunWith(PowerMockRunner.class)
-@PrepareForTest(Jenkins.class)
-@PowerMockIgnore({"com.sun.org.apache.xerces.*", "javax.xml.*", "org.xml.*"})
```

Comment:
Ah yes this mess, best left in the rear-view mirror.

Do we care about adding a Mockito IT? To #441 I guess, so prove that the inline static stuff works?

Reply:
I don't think so =/.

Seems a bit weird to add something like that to me. Powermock was more of a special case even so I'm not sure having it here was the best, it wasn't a magic fix to read it from the parent pom and it worked across java versions, you needed the above mess all over the place.